docs/tutorials/typography.rst

```diff
@@ -175,8 +175,6 @@ Before a font is used in a program, it must be converted and set as the current
 
    from p5 import *
 
-   zigBlack = None
```

Comment:
I think the first two examples should also declare zigBlack / sourceLight "global" in setup, even if it isn't strictly necessary for those example -- that is a good basic expectation, that these should be accessible later if you want them. Learners cut-paste these as boilerplate... and then later they try to use `textFont()` and it doesn't work and they don't understand why.

Reply:
Ok agreed, notice that textFont() is not correct ether, should us text_font() but I understand why its good idea to have declare zigBlack / sourceLight "global", maybe for me is weird having variables that are not used.